@here/harp-mapview/lib/MapView.ts

```diff
@@ -642,14 +629,42 @@ export const MapViewDefaults = {
         typeof window !== "undefined" && window.devicePixelRatio !== undefined
             ? window.devicePixelRatio
             : 1.0,
-    geoCenter: new GeoCoordinates(25, 0, 30000000),
     target: new GeoCoordinates(25, 0),
     zoomLevel: 5,
     tilt: 0,
     heading: 0,
     theme: {}
 };
 
+/**
+ * Parameters for MapView.lookAt
+ */
+export interface LookAtParams {
+    /**
+     * Target/look at point of the MapView.
+     * @note If the given point is not on the ground (altitude != 0) [[MapView]] will do a
+     * raycasting internally to find a target on the ground.
+     * As consequence [[MapView.target]] and [[MapView.zoomLevel]] will nit match the values
+     * that were passed into the [[MapView.lookAt]] method.
+     */
+    target?: GeoCoordLike;
+
+    /**
+     * Zoomlevel of the MapView.
+     */
+    zoomLevel?: number;
```

Comment:
I strongly propose that we support also `distance in meters` in this API.

1) we maintain some feature parity with old `lookAt` which supported `distance`
2) I think removing `distance` from API is wrong, as in _3D_ engine is much more natural for users than some abstract, concept like `zoomLevel` which means nothing to random users.

(Yes, distance in mercator at high latidutes is problematic, but that's another story)

This may be followup patch of course.

Reply:
The `lookAt` API is mainly for users that are used to 2d engines like Tangram or Mapbox and these users don't know the concept of distance.
For 3D "power users" there are other ways, e.g. directly modifying the camera or `MapViewUtils.convertZoomLevelToDistance`

Reply:
Hmm, i think there is potential misunderstanding.

I meant support both `distance` and `zoomLevel`, so we just use `MapViewUtils.convertZoomLevelToDistance` internally in `lookAt`

Anyway, that just means that many of our internal tests have to be rewritten to use zoom level, when they were originally created with distance and that's just more work in order to merge this PR..

Reply:
Yeah, I could add it as well but I think it should be deprecated b/c it's hard to understand. It is not the actual distance of the camera to the target point but:
`the distance of the camera to the target point but clamped to min and max zoomlevel ?!?!`.
Also adding this as deprectated in the new API makes not much sense b/c you could also just use the old API.
Since there is an IBC test failing already I will look into switching them all to zoomLevel b/c this is also what you want to express. We have tests that want to test level 1 but instead of setting zoomLevel to 1 we set it to some weird distance.